tests/templates/kuttl/spark-connect/10-deploy-spark-connect.yaml.j2

```diff
@@ -36,15 +36,29 @@ spec:
   vectorAggregatorConfigMapName: vector-aggregator-discovery
 {% endif %}
   args:
-    - --packages org.apache.iceberg:iceberg-spark-runtime-{{ ".".join(test_scenario['values']['spark-connect'].split('.')[:2]) }}_2.12:1.8.1
+    # These are unfortunatly required to make the S3A connector work with MinIO
+    # I had expected the clients to be able to set these, but that is not the case.
+    - --conf spark.hadoop.fs.s3a.aws.credentials.provider=org.apache.hadoop.fs.s3a.SimpleAWSCredentialsProvider
+    - --conf spark.hadoop.fs.s3a.path.style.access=true
+    - --conf spark.hadoop.fs.s3a.endpoint=http://minio:9000
+    - --conf spark.hadoop.fs.s3a.region=us-east-1
```

Comment:
Just to check, Spark Connect doesn't use these?

- https://crds.stackable.tech/25.3.0/s3.stackable.tech/s3connection/v1alpha1/
- https://crds.stackable.tech/25.3.0/s3.stackable.tech/s3bucket/v1alpha1/